.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: python
 sudo: false
+dist: xenial
 python:
     - "2.7"
-    - "3.5"
     - "3.6"
+    - "3.7"
```

Comment:
These changes in the supported version should also be reflected in `setup.py`.
Additionally `python_requires` would be nice, see https://github.com/zopefoundation/Zope/commit/11321e422801361f52b1bfa885f457567fddbe67

Reply:
Ok, makes sense.